tools/bin/mbedtls-prepare-build

```diff
@@ -1185,6 +1185,25 @@ if [ -n "$$failures" ]; then echo; echo "Failures:$$failures"; false; fi
         self.add_clean('$(test_apps)')
         # TODO: test_psa_constant_names.py
 
+    COVERAGE_RULE = """
```

Comment:
Any particular reason we do not define it as a list of strings?

COVERAGE_RULE= ["command A",
                                "command B",
                                "command C" ]

Rather that a docstring which we proccess in an iterator?

Reply:
LGTM

Reply:
No particular reason. I guess I felt that since it's a long succession of commands with no variables, it was more readable that way. Also it's copied from `Makefile` and makes it easier to reconcile any changes that way.